doc/api/buffer.markdown

```diff
@@ -541,6 +541,42 @@ console.log(bufA.length);
 // 42
 ```
 
+### Class Method: Buffer.encode(str[, str_encoding][, target_encoding])
```

Comment:
Not sure whether there are rules on this, but maybe write `[[, str_encoding], target_encoding]` to avoid ambiguity wrt which parameter is optional?